LICENSE

```diff
@@ -0,0 +1,166 @@
+
+		   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
I don't what the others are thinking, but usually, we tend to use GPLv3 for device servers and LGPL license only for libraries.
This repository does not contain any library so I would suggest to use GPLv3 license instead of LGPL for this repository. This comment would apply to hdbpp-es  (and hdbpp-cm-es) repository as well.

Reply:
I was unaware of this. I will swap it for all hdb++ device servers and update package license.